docs/getting-started/ory-cloud-oauth2.mdx

```diff
@@ -0,0 +1,225 @@
+---
+id: ory-cloud-oauth2
+title: Perform OAuth2 Authorization Code Grant with Ory Cloud
+sidebar_label: OAuth2 in Ory Cloud
+---
+
+This guide shows you a sample implementation of an Authorization Code Grant OAuth2 flow in Ory Cloud that you can use as a
+starting point for using the Ory Cloud OAuth2 Service (Ory Hydra). It takes you on a complete journey, from initial setup to
```

Comment:
In the spirit of cloud-first and concise names, let's call the thing "Ory OAuth2 Federation" for now, and shorten to something like:

This guide shows you a sample implementation of an _Authorization Code Grant_ OAuth2 flow using Ory OAuth2 Federation* in Ory Cloud.

... then I'd like to relegate Ory Hydra to a footnote, as in: 
* Ory OAuth2 Federation is fully managed in Ory Cloud and based on the open source Ory Hydra federation server.

Reply:
Done, please see if the way I did this is okay for you :)